<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dcontrolraton</class>
 <widget class="QDialog" name="dcontrolraton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>111</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control del Rat√≥n</string>
  </property>
  <widget class="QToolButton" name="btn_abajo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>26</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_izquierda">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>26</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_arriba">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>26</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>/\</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_derecha">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>26</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
